library/src/main/java/com/mvc/imagepicker/ImagePicker.java

```diff
@@ -58,29 +60,71 @@
     private static int minWidthQuality = DEFAULT_MIN_WIDTH_QUALITY;
     private static int minHeightQuality = DEFAULT_MIN_HEIGHT_QUALITY;
 
+    private static String mChooserTitle;
+    private static int mPickImageRequestCode = DEFAULT_REQUEST_CODE;
```

Comment:
If you return the same request code for different imageviews on sceens it's not easy to define what exactly imageview need to be changed.